bigcrunch/webapp.py

```diff
@@ -39,13 +39,16 @@ def get_or_create(self):
             try:
                 response = yield from client.create_cluster(
                     ClusterIdentifier=cluster_identifier,
-                    NodeType='dc1.large',
+                    NodeType='dc2.large',
```

Comment:
We'll likely get better start times by using a current instance type since dc1.large are being phased out.